src/flixel/util/FlxList.as

```diff
@@ -36,6 +42,8 @@ package flixel.util
 			if(next != null)
 				next.destroy();
 			next = null;
+
+			listPool.disposeObject(this);
```

Comment:
This line is why trying to destroy the same `FlxList` twice would be disastrous. If it's in the `FlxObjectPool` twice, then two completely different objects could get pointers to it, erroneously thinking they each had a unique `FlxList`